### PR TITLE
Add uninstall profile

### PR DESCRIPTION
**Pull request by hvelarde**
This pull request add an uninstall profile for branch 1.x only; work based on this must be done in master branch to guarantee uninstall is available for Mosaic 2.x and later.

refs. #418
refs. https://github.com/plone/plone.app.standardtiles/pull/90

### CODE REVIEW

File: src/plone/app/mosaic/configure.zcml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:zcml="http://namespaces.zope.org/zcml"
     i18n_domain="plone">
 
+  <five:registerPackage package="." />
```
Comment (jensens) [inline src/plone/app/mosaic/configure.zcml]: there is nothing to load in ``plone/app/mosaic/__init__.py`` so why did you add it?

Comment (hvelarde) [reply src/plone/app/mosaic/configure.zcml]: IIRC, without this the `Extensions` module is not loaded and there's no uninstall then; this is the case for Plone <= 5.0.

Comment (jensens) [reply src/plone/app/mosaic/configure.zcml]: Huh, Extensions. Is this still needed in 4.3? IIRC setuphandlers are fully supported in 4.3 latest?

Comment (hvelarde) [reply src/plone/app/mosaic/configure.zcml]: no, the uninstall profile doen't run automatically in Plone 4.3.
